### DISCUSSION TITLE
How to ignore commands with underscore prefix when auto-completion commands?

### DISCUSSION THREAD

**Discussion by BiosSun** [ohmyzsh/ohmyzsh#10111] [Help]
There are a lots of underscore-prefixed commands in my shell environment (e.q. _stow). I will not use them, but when I use auto-completion commands, they are always Will appear, affecting my normal use.

So is there any way to ignore them when auto-completion commands, they will only appear during auto-completion only when I explicitly type the underscore.

**Comment by Roc25** [comment]
It's too late, but 'HYPHEN_INSENSITIVE="true"' helped me. Now '_stow' doesn't appear when I want to use 'stow'. But it doesn't hide all underscore functions
